### PR TITLE
add explanation about validity and max-allowed-end parameters

### PR DESCRIPTION
**Pull request by QiufangMa**

### CODE REVIEW

**Comment by QiufangMa** [comment]
#11

File: yang/ietf-schedule.yang [hunk 1]
```diff
@@ -448,7 +448,7 @@ module ietf-schedule {
       }
       augment "recurrence-first" {
         description
-          "Add a parameter indicating how long the occurence 
+          "Add a parameter indicating how long the occurrence
```
Comment (boucadair) [inline yang/ietf-schedule.yang:451]:
```suggestion
          "Adds a parameter indicating how long the occurrence 
```

File: yang/ietf-schedule.yang [hunk 2]
```diff
@@ -273,27 +273,27 @@ module ietf-schedule {
...
-         occurence of that new schedule is scheduled before 
+         occurrence of that new schedule is scheduled before 
          'min-allowed-start'.";
     }
     leaf max-allowed-end {
       type yang:date-and-time;
       description
         "A requested schedule will be rejected if the end time of 
-         the last occurence exceed 'max-allowed-end'.";
+         the last occurrence exceed 'max-allowed-end'.";
```
Comment (boucadair) [inline yang/ietf-schedule.yang:296]:
```suggestion
         the last occurrence exceeds 'max-allowed-end'.";
```

File: draft-ietf-netmod-schedule-yang.md [hunk 3]
```diff
@@ -142,14 +142,17 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+   can be executed by a system and takes precedence over similar attributes
+   that are provided at the schedule instance itself.
 
    The "max/min-allowed-start" parameters specify the maximum/minimum scheduled
    start date and time, a requested schedule will be rejected if the first
    occurrence of the schedule is later/earlier than the configured values.
 
    The "max-allowed-end" parameter specifies the maximum allowed end time of
-   the last occurrence.
+   the last occurrence. A requested schedule will be rejected if the last
```
Comment (boucadair) [inline draft-ietf-netmod-schedule-yang.md:154]:
```suggestion
   the last occurrence. A requested schedule will be rejected if the duration of the last
```

**Review by boucadair — CHANGES_REQUESTED** [review]
For the end thing, we need to reason as a function of the duration.

File: draft-ietf-netmod-schedule-yang.md (hunk 3 above)
Comment (QiufangMa) [reply draft-ietf-netmod-schedule-yang.md:154]: I think you mean the schedule needs to finish before the max-allowed-end. Would this be better:
A requested schedule will be rejected if the end time of last occurrence is later than the configured "max-allowed-end" value.

Comment (boucadair) [reply draft-ietf-netmod-schedule-yang.md:154]: deal! Thanks/

**Pull request merged**